test/LeanCode.TestCoverage.Tests/TestCoverageTests.cs

```diff
@@ -1,9 +1,7 @@
-using Xunit;
-
 namespace LeanCode.TestCoverage.Tests;
 
 public class TestCoverageTests
 {
-    [Fact]
+    [Xunit.Fact]
```

Comment:
Why this change?

Reply:
Default rulesets of StyleCop (this is the only project where it is included) favor namespaced `using`s. This was the easiest fix.